.github/ISSUE_TEMPLATE/Support_Question.md

```diff
@@ -15,5 +15,5 @@ assignees: ''
 
 ## Community Resources
 
-[Slack Community Chat](https://powershell.slack.com) - Interactive chat with other PowerShell enthusiasts
+[Slack](https://j.mp/psslack) and [Discord](https://j.mp/psdiscord) Community Chat - Interactive chat with other PowerShell enthusiasts. Both Slack and Discord are bridged via a bot and can seamlessly talk to each other.
```

Comment:
why are we using bitly?

Reply:
The existing link is not the join LinkedIn. It simply links to the Slack space and does not allow new users to join it or view it.

The bitly links are easier to remember and both link to semi-permanent join links for each channel. On occasion these links have had to change, so having the shortlink that can be updated is a lot easier than updating the direct links everywhere they're posted.

If you guys would rather create an aka.ms link, you're more than welcome to! :smile:

I think these links are maintained by @jaykul, to the best of my knowledge.

Reply:
Since these two chat channels are community maintained and not Microsoft, I would not create aka.ms links to them.  if we can confirm that Joel Bennett or some MVP is maintaining those links, I'd be more comfortable using them.

Reply:
Alternatively, we can have the long url at the bottom of the doc and ref the link that way so that the markdown is still readable as just text

Reply:
@stevel-msft so I'm not against that, but the link currently in this document is unusable by anyone who's not already joined the slack space.

If we want to do that, we'd need the actual join link here (which I guess we can get from the shortlink either way).

@jaykul, you're maintaining that shortlink, right?

Reply:
@vexx32 yes, I was suggesting using the resolved long url and to put it at the bottom of the doc to not impede readability since it's not made for human consumption

Reply:
Huh, looks like @TylerLeonhardt beat us to it, I didn't realize there were aka.ms versions already:

https://aka.ms/psdiscord https://aka.ms/psslack

I guess we'll use these instead then?

Reply:
I'd like to get @joeyaiello's take on this

Reply:
I don't think anyone wants to maintain an aka.ms link for this from our side, so the fully qualified links are probably the way to go (anchored at the bottom, as @SteveL-MSFT suggested). It's going to be easier for everyone to spot and fix the breakage here than if they have to track down @TylerLeonhardt to get the aka.ms updated. 

And yeah, we shouldn't ship bit.ly links we don't own for security reasons.